includes/discovery/sensors/state/timos.inc.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+$power_supply_table = end(snmpwalk_cache_numerical_oid($device, 'tmnxChassisPowerSupplyTable', $power_supply_table = array(), 'TIMETRA-CHASSIS-MIB', 'aos', '-OQUsn'));
```

Comment:
@laf while looking at this this morning, wouldn't it really be more efficient to just grab the table rather than issuing X number of individual `snmp_get`'s on each device?